app/src/main/res/values/strings.xml

```diff
@@ -47,6 +47,8 @@
     <string name="menu_page_other_languages">Change language</string>
     <string name="menu_page_find_in_page">Find in article</string>
     <string name="menu_page_add_to_watchlist">Add to Watchlist</string>
+    <string name="menu_page_talk">View talk page</string>
+    <string name="menu_page_edit_history">View edit history</string>
```

Comment:
Do we really need the text "View" instead of showing "Talk page" and "Edit History"?

Reply:
Good catch .. looks like Jazmin has also skipped view on the ticket

Reply:
I have updated it according to the strings Jazmin has put it @cooltey .